browser/src/Services/Explorer/ExplorerFileSystem.ts

```diff
@@ -58,4 +75,77 @@ export class FileSystem implements IFileSystem {
     public exists(fullPath: string): Promise<boolean> {
         return this._fs.exists(fullPath)
     }
+
+    /**
+     * Delete a file or Folder
+     *
+     * @name deleteNode
+     * @function
+     * @param {ExplorerNode} node The file or folder node
+     */
+    public deleteNode = (node: ExplorerNode): void => {
```

Comment:
@DeltaEvo do you think it will make a difference to the functionality here? they point to the right `this` in this scenario

Reply:
Yeah i know, the behavior is not different, the only thing different is that you can't set another this for this function
I was asking just because we are in a class and using class functions seems less "hacky"

Reply:
Tbh using arrow functions in classes is something I do primarily because of all the hiccups with losing the correct this bindings in react class component methods.

Personally when I use an arrow function in a class which is almost paradigmatic in react (to avoid multiple lines of [this binding in constructor functions](https://codeburst.io/good-bad-and-ugly-parts-es6-class-arrow-functions-and-react-1ead4fa2c763)), it's primarily to signal that I intended for a function to be used with the this of where it was defined.

I like to avoid situations where `this` is changed as that for me has lead to more bugs than not.

Reply:
I will vouch that using arrow functions to avoid manual binding of class methods is idiomatic react, FWIW. 😄

Reply:
Uw okay thanks for the explanation, so in this case can you change the three methods that are in this class to use arrow functions too ? :heart: